inc/functions.php

```diff
@@ -39,30 +39,29 @@ function output_page($contents)
             $serverload = get_server_load();
 
             if (my_strpos(getenv("REQUEST_URI"), "?")) {
-                $debuglink = htmlspecialchars_uni(getenv("REQUEST_URI")) . "&amp;debug=1";
+                $debuglink = getenv("REQUEST_URI") . "&amp;debug=1";
```

Comment:
I haven't checked, but this may result in the output being `&amp;amp;` due to the `&` here being escaped by twig...